scoop-basics/src/main/java/com/example/scoop/basics/scoop/ButterKnifeViewBinder.java

```diff
@@ -1,18 +1,23 @@
 package com.example.scoop.basics.scoop;
 
 import android.view.View;
-import butterknife.ButterKnife;
+
 import com.lyft.scoop.ViewBinder;
 
+import butterknife.ButterKnife;
+import butterknife.Unbinder;
+
 public class ButterKnifeViewBinder implements ViewBinder {
 
+    private Unbinder unbinder;
+
     @Override
     public void bind(Object object, View view) {
-        ButterKnife.bind(object, view);
+        unbinder = ButterKnife.bind(object, view);
```

Comment:
There might be more bookkeeping involved here. You probably need to keep an unbinder instance per `object`